### PR TITLE
Copy over new version of Milo's geo-based locales

### PR DESCRIPTION
**Pull request by hparra**
- Adopt milo fix for incorrect lang attributes on html tag
- Add new `reg` property for future use

See https://github.com/adobecom/milo/pull/915 and https://github.com/adobecom/milo/pull/938

**Test URLs:**
- Before: https://main--bacom--adobecom.hlx.page/customer-success-stories?martech=off
- Before: https://main--bacom--adobecom.hlx.page/la/customer-success-stories?martech=off
- After: https://hparra-patch-2--bacom--adobecom.hlx.page/customer-success-stories?martech=off
- After: https://hparra-patch-2--bacom--adobecom.hlx.page/la/customer-success-stories?martech=off

### CODE REVIEW

**Review by JasonHowellSlavin — APPROVED** [review]
LGTM. Alphabetizing for the win!

**Review by meganthecoder — APPROVED** [review]

**Comment by meganthecoder** [comment]
@hparra There's no ui or functionality to verify here, right? It's just adding the new `lang` and `reg` properties and alphabetizing?

**Comment by hparra** [comment]
@meganthecoder There is new functionality in Milo that is enabled when these values exist. You can confirm in test urls that HTML lang attribute now only uses language subtag instead of full tag, e.g `<html lang="en">` vs. `<html lang="en-US">`. See https://github.com/adobecom/milo/pull/915 and linked issue for more details. Milo features use `getConfig().locale` and do not read locale from this attribute.

**Comment by meganthecoder** [comment]
@hparra Cool, I updated the description to reflect that and assigned myself to verify. Is the `reg` tag being used yet? I didn't see anything in the PRs you mentioned.

**Comment by hparra** [comment]
> Is the `reg` tag being used yet?

Not yet. A couple of new features use it and I will begin porting existing features to use it one by one -- those changes will be in Milo only but it will affect all Milo consumers. There will be additional comms of those.

File: scripts/scripts.js [hunk 1]
```diff
@@ -35,89 +35,99 @@ const CONFIG = {
...
+    ch_it: { ietf: 'it-CH', lang: 'it', reg: 'CH', tk: 'bbf5pok.css' },
+    cl: { ietf: 'es-CL', lang: 'es', reg: 'CL', tk: 'oln4yqj.css' },
+    cn: { ietf: 'zh-CN', lang: 'zh', reg: 'CN', tk: 'puu3xkp' },
+    co: { ietf: 'es-CO', lang: 'es', reg: 'CO', tk: 'oln4yqj.css' },
+    cr: { ietf: 'es-419', lang: 'es', reg: 'CR', tk: 'oln4yqj.css' },
+    cy_en: { ietf: 'en-CY', lang: 'en', reg: 'CY', tk: 'pps7abe.css' },
+    cz: { ietf: 'cs-CZ', lang: 'cs', reg: 'CZ', tk: 'aaz7dvd.css' },
+    de: { ietf: 'de-DE', lang: 'de', reg: 'DE', tk: 'vin7zsi.css' },
+    dk: { ietf: 'da-DK', lang: 'da', reg: 'DK', tk: 'aaz7dvd.css' },
+    ec: { ietf: 'es-419', lang: 'es', reg: 'EC', tk: 'oln4yqj.css' },
```
Comment (meganthecoder) [inline scripts/scripts.js:63]: Same for ec (ecuador)
```suggestion
```

File: scripts/scripts.js [hunk 2]
```diff
@@ -35,89 +35,99 @@ const CONFIG = {
...
+    ee: { ietf: 'et-EE', lang: 'et', reg: 'EE', tk: 'aaz7dvd.css' },
+    eg_ar: { ietf: 'ar', lang: 'ar', reg: 'EG', tk: 'nwq1mna.css', dir: 'rtl' },
+    eg_en: { ietf: 'en-GB', lang: 'en', reg: 'EG', tk: 'pps7abe.css' },
+    el: { ietf: 'el', lang: 'el', reg: 'GR', tk: 'aaz7dvd.css' },
+    es: { ietf: 'es-ES', lang: 'es', reg: 'ES', tk: 'oln4yqj.css' },
+    fi: { ietf: 'fi-FI', lang: 'fi', reg: 'FI', tk: 'aaz7dvd.css' },
+    fr: { ietf: 'fr-FR', lang: 'fr', reg: 'FR', tk: 'vrk5vyv.css' },
+    gr_el: { ietf: 'el', lang: 'el', reg: 'GR', tk: 'fnx0rsr.css' },
+    gr_en: { ietf: 'en-GR', lang: 'en', reg: 'GR', tk: 'pps7abe.css' },
+    gt: { ietf: 'es-419', lang: 'es', reg: 'GT', tk: 'oln4yqj.css' },
```
Comment (meganthecoder) [inline scripts/scripts.js:73]: Same for gt (guatemala)
```suggestion
```

File: scripts/scripts.js [hunk 3]
```diff
@@ -35,89 +35,99 @@ const CONFIG = {
...
+    my_ms: { ietf: 'ms', lang: 'ms', reg: 'MY', tk: 'sxj4tvo.css' },
+    ng: { ietf: 'en-GB', lang: 'en', reg: 'NG', tk: 'pps7abe.css' },
+    nl: { ietf: 'nl-NL', lang: 'nl', reg: 'NL', tk: 'cya6bri.css' },
+    no: { ietf: 'no-NO', lang: 'no', reg: 'NO', tk: 'aaz7dvd.css' },
+    nz: { ietf: 'en-GB', lang: 'en', reg: 'NZ', tk: 'pps7abe.css' },
+    pe: { ietf: 'es-PE', lang: 'es', reg: 'PE', tk: 'oln4yqj.css' },
+    ph_en: { ietf: 'en', lang: 'en', reg: 'PH', tk: 'pps7abe.css' },
+    ph_fil: { ietf: 'fil-PH', lang: 'fil', reg: 'PH', tk: 'ict8rmp.css' },
+    pl: { ietf: 'pl-PL', lang: 'pl', reg: 'PL', tk: 'aaz7dvd.css' },
+    pr: { ietf: 'es-419', lang: 'es', reg: 'PR', tk: 'oln4yqj.css' },
```
Comment (meganthecoder) [inline scripts/scripts.js:110]: Same for pr (puerto rico)
```suggestion
```

**Review by meganthecoder — CHANGES_REQUESTED** [review]
There are a few locales added that we don't support in bacom. Also, when I was testing, I saw that CaaS cards weren't loading for `la`

**Comment by hparra** [comment]
> There are a few locales added that we don't support in bacom

This should be OK. `locales` is just a source of truth for mappings. `locales` should NOT be used to enable or disable a particular locale. I can remove but it makes syncing more difficult.

> Also, when I was testing, I saw that CaaS cards weren't loading for `la`

This is probably not OK and is the sort of thing I was afraid of. Can we sync on this so I can understand what is happening?

**Comment by hparra** [comment]
Waiting for https://github.com/adobecom/milo/pull/945

**Comment by JasonHowellSlavin** [comment]
@hparra @meganthecoder looks like `la` cards are loading again on this one, some links: 
https://hparra-patch-2--bacom--adobecom.hlx.page/la/customer-success-stories?martech=off
https://hparra-patch-2--bacom--adobecom.hlx.page/la/resources/main

**Comment by meganthecoder** [comment]
I believe a different solution was determined here: https://github.com/adobecom/milo/pull/963

@hparra Can this PR be closed?

**Comment by hparra** [comment]
@meganthecoder I was waiting until handoff to close all PRs but I think we're ok.

**Pull request closed without merging**